src/map/utils/fishingutils.cpp

```diff
@@ -2891,8 +2891,11 @@ namespace fishingutils
 
         if (PItem && FishList[itemid])
         {
-            // CItemFish constructor uses `const CItem&` input so this is ok
-            return new CItemFish(*PItem);
+            if (CItemUsable* PItemUsable = dynamic_cast<CItemUsable*>(PItem))
```

Comment:
What is the purpose of this cast?
 `class CItemEquipment : public CItemUsable` 
Neither of these classes are parents or children of CItemFish
What problem were you trying to solve here?

This is weird because I know (some) fish can be eaten by mithras or with some hidden effect items. Maybe CItemFish should be a child of CItemUsable..?

Reply:
Yeah I was trying something with casting and forgot to remove this piece.  I've taken this out and am resetting it back to what is on LSB.  I'm just not even using the GetFish function and instead checking the exdata directly.

Reply:
Oh ok, makes sense. I still wonder what's going on with usable fish then... hmm...